### ISSUE TITLE
Utf-8 escaped in 0.9.3

### ISSUE THREAD

**Issue by Looted** [rails-api/active_model_serializers#1078]
Hey,
for whatever reason unicode characters get escaped when using AMS 0.9.3 (Rails 3.2.22, Oj). Is this desired behaviour? Is there any way I can disable escaping them?

**Comment by Looted** [comment]
Turns out I wasn't using Oj after all (despire MultiJson.use(:oj) in initializer). What I needed was https://github.com/GoodLife/rails-patch-json-encode. Unicode content doesn't get escaped now as it should.

**Comment by joaomdmoura** [comment]
Cool @Looted , glad that you found your way through it. Btw you should give it a try to 0.10.0rc2 or master :smile:

**Comment by Looted** [comment]
Is the rc or master production ready? I settled for stable version for now, but I'm really excited about caching in 0.10 :)

**Comment by joaomdmoura** [comment]
Yup, there are some ppl already using it on production, me included (despite of not being using the latest version).
Our new version is schedule to be released on the end of Octuber.

**Comment by timwaters** [comment]
I'm using 10.0.rc5 (and rails 4 and default JSON) and am seeing this issue with pagination links in the json-api adapter. 
e.g `"last":"http://test.host/api/v1/maps.json?foo=bar\u0026page%5Bnumber%5D=2\u0026page%5Bsize%5D=1`

In the tests for rails-api the expected response for pagination links in the json-api adapter is one without unicode encoded characters. 

I have tried using the rails-patch-json-encode gem, as mentioned above but I see no difference in the output. 

One way that I have managed to not have unicode escaped character for ` &` is to set 
`ActiveSupport.escape_html_entities_in_json = false` which is probably not an advised workaround.
